DOCKER_README.MD

```diff
@@ -0,0 +1,79 @@
+In this page you will find our recommended way of installing Docker on your machine. 
+This guide is made for OSX users.
+
+## Install docker
```

Comment:
Can you apply the suggested changes in the other repos on this README, please? E.g. `Docker` is not capitalized everywhere in this file. Thank you 😄